tests/test_charms.py

```diff
@@ -304,8 +300,7 @@ async def test_prometheus_grafana_integration(ops_test):
 
     for attempt in retry_for_5_attempts:
         logger.info(
-            f"Testing prometheus deployment (attempt "
-            f"{attempt.retry_state.attempt_number})"
+            f"Testing prometheus deployment (attempt " f"{attempt.retry_state.attempt_number})"
```

Comment:
I would have reformatted it as:
`f"Testing prometheus deployment (attempt {attempt.retry_state.attempt_number})"`